### DISCUSSION TITLE
如何有效查看租户新增功能的程序调用逻辑

### DISCUSSION THREAD

**Discussion by WZR8277** [zuihou/zuihou-admin-cloud#227] [02.提问]
![image](https://user-images.githubusercontent.com/68729543/204084030-654205c0-a138-4753-9b83-f8bd02101d07.png) 请问我在新增租户时，在红线的对应函数打断点，但是新增时并没有进入断点，数据库也新增成功了是为什么呢？我的目的是查看租户新增功能的程序调用关系，梳理逻辑。

**Comment by zuihou** [comment]
1. 确保是debug模式启动
2. 浏览器F12控制台打开，点新增时，确保请求的是 post  /api/xx/tenant 接口

**Reply by WZR8277** [reply]
多谢
